projects/rero/ng-core/src/lib/record/editor/type/remote-typeahead/remote-typeahead.component.ts

```diff
@@ -52,10 +53,9 @@ export class RemoteTypeaheadComponent extends FieldType implements OnInit {
   /**
    * Constructor
    * @param _remoteTypeaheadService - RemoteTypeaheadService
+   * @param _route - Activated route
    */
-  constructor(
-    private _remoteTypeaheadService: RemoteTypeaheadService
-  ) {
+  constructor(private _remoteTypeaheadService: RemoteTypeaheadService, private _route: ActivatedRoute) {
```

Comment:
Seems better to have each constructor argument on separate line.

Reply:
`Prettier` says no...